Makefile

```diff
@@ -186,12 +186,29 @@ run-test-multikueue-e2e-%: FORCE
 	@echo Running multikueue e2e for k8s ${K8S_VERSION}
 	E2E_KIND_VERSION="kindest/node:v$(K8S_VERSION)" KIND_CLUSTER_NAME=$(KIND_CLUSTER_NAME) CREATE_KIND_CLUSTER=$(CREATE_KIND_CLUSTER) ARTIFACTS="$(ARTIFACTS)/$@" IMAGE_TAG=$(IMAGE_TAG) ./hack/multikueue-e2e-test.sh
 
+PROJECT_BIN := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))/bin
+.PHONY: shell-lint
+shell-lint: | $(PROJECT_BIN)
+	cd $(PROJECT_BIN); \
+	if [ ! -d "shellcheck" ]; then \
+		DOWNLOAD_URL=$$(curl -s https://api.github.com/repos/koalaman/shellcheck/releases/latest | grep "browser_download_url.*linux.x86_64.tar.xz" | cut -d : -f 2,3 | tr -d \" ); \
```

Comment:
Should we statically set the download URL, not dynamically?
I meant like the following:

```
GOOS = $(shell go env GOOS)
GOARCH = $(shell go env GOARCH)

...

ifeq ($(GOOS),arm64)
OS = aarch64
else
OS = $(GOOS)
endif
shell-lint:
    wget -q -O /bin/shellcheck.tar.gz https://github.com/koalaman/shellcheck/releases/download/v0.9.0/shellcheck-v0.9.0.$(OS).$(GOARCH).tar.xz
```

Reply:
Sorry for the late response. I tried creating the URL statically 
This is the output of GOOS and GOARCH on my computer
```
>> go env GOOS  
linux
>> go env GOARCH
amd64
```
This is the URL I got but it didn't download the binary as we expected.

```
https://github.com/koalaman/shellcheck/releases/download/v0.9.0/shellcheck-v0.9.0.linux.amd64.tar.xz
```

I'll try something like this
```

GOOS = $(shell go env GOOS)
GOARCH = $(shell go env GOARCH)

ifeq ($(GOOS),darwin)
	URL := https://github.com/koalaman/shellcheck/releases/download/v0.9.0/shellcheck-v0.9.0.darwin.x86_64.tar.xz
else
	ARCH = aarch64
	ifeq ($(GOARCH),arm64)
	else
		ARCH = armv6hf
	endif
	# another condition
endif

.PHONY: shell-lint
shell-lint:
	wget -q -O /bin/shellcheck.tar.gz $(URL)
```

Reply:
I have pushed some changes downloading the release binary statically.